crates/evm/src/executor/snapshot.rs

```diff
@@ -32,10 +32,10 @@ impl<T> Snapshots<T> {
         let snapshot = self.snapshots.remove(&id);
 
         // revert all snapshots taken after the snapshot
-        let mut to_revert = id + 1;
+        let mut to_revert = id.add(U256::from(1));
         while to_revert < self.id {
             self.snapshots.remove(&to_revert);
-            to_revert = to_revert + 1;
+            to_revert = to_revert.add(U256::from(1));
```

Comment:
just use `+` operator?

Reply:
Done in https://github.com/foundry-rs/foundry/pull/5768/commits/2b48fb1f7546576a0bc239067685c0e7d75259a5